src/styles/blocks/_table.scss

```diff
@@ -0,0 +1,45 @@
+table {
+  text-align: left;
+  width: 100%;
+
+  tr {
+    border-bottom: 1px solid var(--gcds-border-default);
```

Comment:
Can you use `border-block-end` instead of `border-bottom` in order to have better multi-language support?